src/rt/msvc_math.c

```diff
@@ -0,0 +1,52 @@
+/**
+* This module provides alternate implementations of single-precision math
+* functions missing in at least some 32-bit x86 MS VC runtimes
+*
+* Copyright: Copyright Digital Mars 2015.
+* License: Distributed under the
+*      $(LINK2 http://www.boost.org/LICENSE_1_0.txt, Boost Software License 1.0).
+*    (See accompanying file LICENSE)
+* Source:    $(DRUNTIMESRC rt/_msvc_math.c)
```

Comment:
But there are no other math modules, contrary to sections and deh. To me, this extra module is part of `msvc.c` and only a separate file to help the linker somewhat.